extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -1338,6 +1338,15 @@ public static enum ApplicationType {
     @ConfigItem
     public Optional<Provider> provider = Optional.empty();
 
+    /**
+     * Indirectly verify the user authentication has been successful with request to `user-info-path`.
+     * Bearer token is considered valid if `user-info-path` endpoint responded with `200 OK`.
+     * You should only enable this option with OIDC non-compliant provider without introspection endpoint.
+     * Please make sure 'user-info' endpoint is secured.
```

Comment:
We can probably remove this line - the access token posted to the UserInfo endpoint is what is used to find a correct UserInfo response. Without the valid access token the provider will never be able to find what to return